go.mod

```diff
@@ -3,8 +3,8 @@ module github.com/crossplane/provider-rook
 go 1.13
 
 require (
-	github.com/crossplane/crossplane v0.13.0-rc.0.20200901163614-365a7c3ad900
-	github.com/crossplane/crossplane-runtime v0.9.1-0.20200831142237-1576699ee9ac
+	github.com/crossplane/crossplane v0.13.0-rc.0.20200923162121-6e6cee50d87f
```

Comment:
I think this can be dropped if https://github.com/crossplane/provider-rook/blob/f29577f3f8ecaaea18c077f1bca3155dab2c632c/cmd/provider/main.go#L120 is removed, which we shouldn't need to register any Crossplane types now